test/master-playlist-controller.test.js

```diff
@@ -3582,6 +3585,80 @@ QUnit.test('creates source buffers after second trackinfo if demuxed', function(
 
 });
 
+QUnit.test('Uses audio codec from audio playlist for demuxed content', function(assert) {
+  const done = assert.async();
+
+  window.MediaSource.isTypeSupported = (type) => (/(mp4a|avc1)/).test(type);
+
+  this.requests.length = 0;
+  this.player.dispose();
+  this.player = createPlayer();
+  this.player.src({
+    src: 'manifest/dash-demuxed.mpd',
```

Comment:
By the DASH interop guidelines, "only non-multiplexed Representations are supported, i.e. each Representation only con39 tains a single media component," so we probably shouldn't specify demuxed in the name of the manifest.